lib/containers/artist.js

```diff
@@ -22,14 +22,19 @@ const TABS = {
 };
 
 class Artist extends React.Component {
-  constructor() {
+  constructor(props) {
     super();
-    this.state = { selectedTabIndex: 1 };
 
     this.availableTabs = this.availableTabs.bind(this);
     this.selectedTabTitle = this.selectedTabTitle.bind(this);
     this.tabSelectionDidChange = this.tabSelectionDidChange.bind(this);
     this.viewForSelectedTab = this.viewForSelectedTab.bind(this);
+
+    if (this.availableTabs().contains(TABS.WORKS)) {
```

Comment:
This doesn't work because `availableTabs` relies on this.props and they're undefined at this points. I know I can get them passed into the constructor but I can't pass them to `availableTabs`.